RecoMuon/MuonSeedGenerator/src/MuonSeedOrcaPatternRecognition.cc

```diff
@@ -118,22 +135,57 @@ void MuonSeedOrcaPatternRecognition::produce(const edm::Event& event, const edm:
   bool* MB2 = zero(list7.size());
   bool* MB3 = zero(list6.size());
 
+  MuonRecHitContainer muRH_GE11Fwd, muRH_GE21Fwd, muRH_GE11Bwd, muRH_GE21Bwd, muRH_ME0Fwd, muRH_ME0Bwd;
+  
+  if (gemForwardLayers.size()){// Forward (z>0) EndCap disk
+    const DetLayer* GE11Fwd = gemForwardLayers[0];
+    muRH_GE11Fwd = filterSegments(muonMeasurements->recHits(GE11Fwd,event), endcapdThetaCut);
+    if (gemForwardLayers.size() > 2){
+      const DetLayer* GE21Fwd = gemForwardLayers[3];
```

Comment:
what is the reason this index is 3 instead of 2, which I'd naively expect compared to the GE11 case that uses 0